server/app/schemas/calculation.post.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  type: "object",
+  required: ["name", "description", "deprecated"],
+  properties: {
+    name: {
+      type: "string",
+      minLength: 1,
+      pattern: "^[a-zA-Z '.-]+$"
```

Comment:
What will happen in the UX if the user breaks validation?

Reply:
I think the create, update , delete, calculation endpoints are [not used](https://github.com/search?q=repo%3Ahackforla%2Ftdm-calculator+calculation+path%3A%2F%5Eclient%5C%2Fsrc%5C%2F%2F&type=code&p=1) by the client currently - future implementation. I think we mostly use the GET endpoints. Also, there is only '1' calculation in the db currently.

Reply:
Then how do you know if this validation is right?

Reply:
The answer could be you are making stubs, but then the code needs a comment that the regex are not spec'ed. 

I am looking for answers to why in the code.

Reply:
And unused endpoints are risky. I am inclined to delete dead code as a security risk. 

Is there a plan for when to add it?

Reply:
excellent question. Ill have to refer to @entrotech

not to mention validation logic may even require numbers to be valid (not included currently)